modules/core/test/test_mat.cpp

```diff
@@ -1248,6 +1248,46 @@ static void check_ndim_shape(const cv::Mat &mat, int cn, int ndims, const int *s
         EXPECT_EQ(mat.size[i], sizes[i]);
 }
 
+TEST(Core_Mat, convertTo)
```

Comment:
Split test cases to different with proper naming:
```
TEST(Core_Mat, convertTo_inplace_8UC3_8SC3)
```
```
TEST(Core_Mat, convertTo_inplace_8UC3_32FC3)
```

Reply:
Yes, I can.